pallets/utility/src/lib.rs

```diff
@@ -125,6 +129,33 @@ impl<C> UniqueCall<C> {
     }
 }
 
+fn combine_classes(mut classes: impl Iterator<Item = DispatchClass>) -> DispatchClass {
+    let all_operational = classes.all(|class| class == DispatchClass::Operational);
+    if all_operational {
+        DispatchClass::Operational
+    } else {
+        DispatchClass::Normal
+    }
+}
+
+fn batch_weight(calls: &[impl GetDispatchInfo]) -> (Weight, DispatchClass) {
+    let weight = calls
+        .into_iter()
+        .map(|call| call.get_dispatch_info().weight)
+        .fold(550_000_000, |a: Weight, n| a.saturating_add(n))
+        .saturating_add(10_000_000);
```

Comment:
NOTE: the `saturating_add(10_000_000)` used to be inside the `fold`, but that is contrary to what the actual documentation of the `batch` function said, so I've moved it out to reflect the docs.

Reply:
Agreed - this looks like it was fixed upstream in https://github.com/paritytech/substrate/pull/5787